tests/specs/session-replay/harvest.e2e.js

```diff
@@ -1,8 +1,25 @@
+import { testRumRequest } from '../../../tools/testing-server/utils/expect-tests'
 import { config, getSR } from './helpers'
 
 export default (function () {
   describe('Session Replay Harvest Behavior', () => {
+    beforeEach(async () => {
+      await browser.testHandle.scheduleReply('bamServer', {
+        test: testRumRequest,
+        body: JSON.stringify({
+          stn: 1,
+          err: 1,
+          ins: 1,
+          cap: 1,
+          spa: 1,
+          loaded: 1,
+          sr: 1
+        })
+      })
+    })
+
     afterEach(async () => {
+      await browser.testHandle.clearScheduledReplies('bamServer')
```

Comment:
The test handle is always destroyed after each test. What was the issue being seen that prompted the creating of this method `clearScheduledReplies`?

Reply:
Wasn't sure if needed tbh (for a test case that might not actually call the RUM call, was not sure what would happen to the scheduled reply)

The actual issue that first prompted was that I wanted to schedule a reply for every test (since nearly all SR tests need the SR: 1 value, which is 0 by default), except one, so for that I just put into a beforeEach, then clear the scheduled response for the 1 test 

https://github.com/newrelic/newrelic-browser-agent/pull/571/files/b116fe95a8b30ca0cdcf42363daee1b8b677af66..766ac0a02dce62607d52435132cef644a7f36fc4#diff-68f6284501cbb642ecebf419e8a964f8dbff379ebc68af9eeb639b8dc4fc4ef6R29-R41

We can do some more clean up if needed

Reply:
Ah ok. Not against it. Just was curious if there was an issue with handles getting stuck in memory or something. You could use a function at the bottom of the test file that enables the reply and only call it in the tests that needed it. No difference to me though.